README.md

```diff
@@ -0,0 +1,16 @@
+# Dogebooook
+Dogebook is a Symfony application created to learn the ins and out of Symfony on the idea of create Dogs' Facebook.
+## Environments
+- Local: [127.0.0.1:8000/](https://127.0.0.1:8000/)
+## Installation
+
+### Requirements
```

Comment:
I'd add commands here to install dependencies like `composer install` 

This could be something you could share with a junior you work with in the future. You shouldn't assume people know how to do things, even the basics.

Reply:
Right will do.